agent/tcs/client/client.go

```diff
@@ -127,9 +127,6 @@ func (cs *clientServer) Serve() error {
 
 // Close closes the underlying connection.
 func (cs *clientServer) Close() error {
-	if cs.publishTicker != nil {
```

Comment:
Not blocking: Should we also remove `publishTicker` from `clientServer` struct?

Reply:
made the change